dist/Dockerfile

```diff
@@ -13,6 +13,6 @@ COPY . .
 RUN cargo build --release
 
 FROM centos
-ENV RUST_LOG=actix_web=error
+ENV RUST_LOG=actix_web=error,dkregistry=error
```

Comment:
We should try to integrate this with the `-v` flag instead of relying on environment variables. This is okay for now though.